src/main/native/cpp/view/AnalyzerPlot.cpp

```diff
@@ -278,4 +327,28 @@ void AnalyzerPlot::DisplayTimeDomainPlots() {
       }
     }
   }
+  if (m_fitNextPlot[6]) {
+    ImPlot::SetNextPlotLimits(0, m_dtMaxTime, 0, 50);
+    ImPlot::FitNextPlotAxes(true, false, false, false);
```

Comment:
@prateekma  is this what you were asking for?

Reply:
Yes

Reply:
Is it good to merge then?

Reply:
Yes, I'll review again tonight to make sure but it should be good to go.